src/main/webapp/WEB-INF/tags/module/purap/preqWireTransfer.tag

```diff
@@ -131,12 +131,12 @@
                     </td>
                 </tr>
                 <tr>
-                    <th scope=row class="bord-l-b"><div align="right"><kul:htmlAttributeLabel attributeEntry="${wireTransAttributes.preqBankSWIFTCode}"/></div></th>
+                    <th scope=row class="bord-l-b"><div align="right"><kul:htmlAttributeLabel attributeEntry="${wireTransAttributes.preqBankSwiftCode}"/></div></th>

```

Comment:
Should the references to `preqBankSwiftCode` be replaced with `bankSwiftCode` instead?

You may also want to check if there are any other property references that need the `preq` prefix removed.

Reply:
@nkimble is this the comment that prompted the latest changes? I think this tag was just supposed to be removed.

Reply:
Lets talk off-line. There are some things going on with these tags.